mod_reforged/hooks/config/strings.nut

```diff
@@ -1338,7 +1345,7 @@ foreach (vanillaDesc in vanillaDescriptions)
 				Type = ::UPD.EffectType.Passive,
 				Description = [
 					"Always start battles at Confident [morale.|Concept.Morale]",
-					::MSU.Text.colorPositive("50%") + " increased [Resolve|Concept.Resolve] against negative morale checks."
+					"Gain " ::MSU.Text.colorPositive("50%") + " increased [Resolve|Concept.Resolve] against negative morale checks."
```

Comment:
`[Resolve|Concept.Bravery] is increased by " + ::MSU.Text.colorPositive("50%") + " against negative [morale checks.|Concept.Morale]"`